### DISCUSSION TITLE
xyz-layer

### DISCUSSION THREAD

**Discussion by jukos** [versatiles-org/versatiles-documentation#30] [General]
Hello,
is there an https address so that I can create an xyz-layer with QGIS?
I can easily produce my own tiles in QGIS and integrate them locally.
Many thanks and best regards

**Comment by MichaelKreil** [comment]
like this: https://github.com/versatiles-org/versatiles-documentation/blob/main/guides/use_versatiles_in_qgis.md ?

**Reply by jukos** [reply]
Thank you very much. It worked.
If I have any further questions, I'll ask a new question.
Best regards
